src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -239,10 +239,12 @@ public function getReadInfo(string $class, string $property, array $context = []
         foreach ($this->accessorPrefixes as $prefix) {
             $methodName = $prefix.$camelProp;
 
-            if ($reflClass->hasMethod($methodName) && ($reflClass->getMethod($methodName)->getModifiers() & $this->methodReflectionFlags)) {
+            if ($reflClass->hasMethod($methodName)) {
```

Comment:
Couldn't you add the check for existing parameters directly in this `if`?

Reply:
If I inline all conditions on a single line, the line is really long. But if you want, no problem!

Reply:
Yes please 👍🏻

Reply:
Done